cluster-state-service/handler/api/v1/taskapis.go

```diff
@@ -24,19 +24,22 @@ import (
 	"github.com/blox/blox/cluster-state-service/handler/store"
 	"github.com/blox/blox/cluster-state-service/handler/types"
 	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
 )
 
 const (
 	taskARNKey     = "arn"
 	taskClusterKey = "cluster"
 
-	taskStatusFilter  = "status"
-	taskClusterFilter = "cluster"
+	taskStatusFilter    = "status"
+	taskClusterFilter   = "cluster"
+	taskStartedByFilter = "startedBy"
```

Comment:
I would suggest that requiring mixed case query string parameters is an invitation for error. While the w3c specs say query strings are case-sensitive, URLs are often typed or read and transcribed, so avoiding case-sensitivity is a kindness to the user.